build-logic/convention/src/main/kotlin/AndroidHiltConventionPlugin.kt

```diff
@@ -7,13 +7,13 @@ class AndroidHiltConventionPlugin : Plugin<Project> {
     override fun apply(target: Project) {
         with(target) {
             with(pluginManager) {
-                apply("com.google.devtools.ksp")
+                apply("org.jetbrains.kotlin.kapt")
```

Comment:
kapt is being depreciated, we need to stick with `ksp`

Reply:
revert to ksp

Reply:
yes sir, but we didn't use ksp,  we created plugin but we used kapt because  DbFlow library used kapt for generating file so we used kapt